instrumentation/rpc/README.md

```diff
@@ -1,32 +1,86 @@
 # brave-instrumentation-rpc
 
-This is a helper for RPC libraries such as gRPC and Dubbo. Specifically, this
-includes samplers for clients and servers, configured with `RpcTracing`.
+Most instrumentation are based on rpc communication. For this reason,
```

Comment:
not sure why RPC was `lowercase`d but maybe good to `uppercase` it as they don't represent a word?

Reply:
I'll sweep good idea